changelog/current.md

```diff
@@ -37,3 +38,6 @@
 - In `c4/charconv.hpp`: do not use C4_ASSERT in `to_c_fmt()`, which is `constexpr`.
 
 
+### Thanks
+
+- @cschreiber
```

Comment:
Thanks! That would be "cschreib" to get the right GitHub ref :)

Reply:
Fixed it!